### ISSUE TITLE
Add possibility to select different methods of normalization in the left menu

### ISSUE THREAD

**Issue by dharasim** [DCMLab/MIDFT#36]
Currently, we are normalizing using the 0th coefficient (i.e., the sum of the input pitch class vectors).
Jason suggested also to try out normalizing by signal power.

**Comment by fabianmoss** [comment]
Hm... I think that this is very technical and not super useful for the user to switch between different normalization methods. But if there are good reasons to use the one mentioned by Jason (and used by Emmanuel), we could hard-code it.

**Comment by dharasim** [comment]
Yes, let's try it out first. In theory, the other normalization method should lead to traces that are not as squeezed together for large window lengths.

**Comment by fabianmoss** [comment]
Maximum entropy (= log_2(12)) might also be useful a normalizing constant

**Comment by heyGio** [comment]
53362338ead11ca40fbbeecd0a8fbd683d355eaa
